src/ucp/stream/stream_recv.c

```diff
@@ -532,8 +532,8 @@ ucp_stream_am_handler(void *am_arg, void *am_data, size_t am_length,
                                   "stream data");
     ep_ext = ucp_ep_ext_proto(ep);
 
-    if (ucs_unlikely(ep->flags & (UCP_EP_FLAG_CLOSED |
-                                  UCP_EP_FLAG_FAILED))) {
+    if (ucs_unlikely((ep == NULL) ||
```

Comment:
why changed here?

Reply:
merge issue, it was done before `UCP_WORKER_GET_VALID_EP_BY_ID` was introduced, BTW, should not we also drop on failed EP there?